### ISSUE TITLE
Issue with converting typescript to javascript

### ISSUE THREAD

**Issue by avinashupadhya99** [avinashupadhya99/TNB-Discord-Security#1]
Running the compiled javascript code results in the following error with `  "type": "module"` in *package.json*- 
```
TNB-Discord-Security/prod/index.js:30
Object.defineProperty(exports, "__esModule", { value: true });
                      ^

ReferenceError: exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and 'TNB-Discord-Security\package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at TNB-Discord-Security/prod/index.js:30:23
    at ModuleJob.run (node:internal/modules/esm/module_job:183:25)
    at async Loader.import (node:internal/modules/esm/loader:178:24)
    at async Object.loadESM (node:internal/process/esm_loader:68:5)
    at async handleMainPromise (node:internal/modules/run_main:63:12)
```

Running the compiled javascript code results in the following error without `  "type": "module"` in *package.json*- 
```
(node:806) UnhandledPromiseRejectionWarning: /mnt/c/Users/avina/Projects/TNB-Discord-Security/src/data/entities/key.entity.ts:1
import {Column, Entity, PrimaryGeneratedColumn} from 'typeorm';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:988:16)
    at Module._compile (internal/modules/cjs/loader.js:1036:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at /mnt/c/Users/avina/Projects/TNB-Discord-Security/node_modules/typeorm/util/DirectoryExportedClassesLoader.js:42:39
    at Array.map (<anonymous>)
    at Object.importClassesFromDirectories
```
    
The code works fine with `npm run dev` but that uses `ts-node-dev` and cannot be used for production

**Comment by johnbabu021** [comment]
Hey @avinashupadhya99  it says change the file extension to cjs

**Comment by johnbabu021** [comment]
.cjs

**Comment by illusi0nary** [comment]
As we spoke i added a pull request for fixing the issue :-)

**Comment by avinashupadhya99** [comment]
Closed in #3
